.github/workflows/release.yaml

```diff
@@ -0,0 +1,121 @@
+name: New release
+on:
+  push:
+    branches:
+      - timbod-githubci
```

Comment:
timbod-githubci branch mentioned - migrate to master for landing it

Reply:
I think running release.yaml on tagging would be good.
https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#example-including-branches-and-tags

Reply:
Yep. That was my intention. Got lost in a weird github push problem.

Reply:
Fixed.